### ISSUE TITLE
Color picker incorrectly remembers/applies info from last use of color picker

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#1741]
1. New HTML file
2. Drag two Content Pane's onto canvas
3. Select first content pane and change background color using color picker dialog
4. Select second content pane and bring up color picker in background-color field. Last value is remembered (which isn't so bad), but it also automatically applies the color chosen for first content pane to 2nd content pane (updating canvas) and cancelling color picker doesn't undo this change.

Environment: Chrome, locally installed on windows

**Comment by JonFerraiolo** [comment]
I just tried to reproduce and I couldn't reproduce on Mac/Chrome. When I selected the second ContentPane and brought up the color picker for the second time, the color picker came up with #ffffff as the currently selected color, and if I click on cancel, no color value is applied to second ContentPane. I expect there is a bug in there somewhere, but maybe it requires a more complicated scenario to reproduce.
